src/univariate/continuous/ksdist.jl

```diff
@@ -16,52 +16,52 @@ function cdf(d::KSDist,x::Float64)
     n = d.n
     b = x*n
     # known exact values
-    if b <= 0.5
-        return 0.0
-    elseif b <= 1.0
+    if b <= 1/2
+        return 0
```

Comment:
See above. This distribution has no parameters, so we didn't have any type instabilities to worry about.